SelfReflection/SelfReflection/Assets/Scripts/Raycast/MoveObjectController.cs

```diff
@@ -0,0 +1,150 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MoveObjectController : MonoBehaviour
+{
+    [Header("Pickup Settings")]
+    private InteractableObject interactableObject;
+    public int interactableLayerIndex;
+
+    [Header("Physics Parameters")]
+    public RaycastHit hit;
+    private Ray ray;
+    public float maxDistance = 150f;
+    public int maxReflections;
+    private float mouseX, mouseY, mouseScroll;
+    public float sensX;
+    public float sensY;
+    public float mouseScrollSense;
+    public float objectMoveSpeed;
+
+    private Transform relativeMirror;
+
+    private void Start()
+    {
+        sensX = transform.GetComponent<PlayerCam>().sensX;
+        sensY = transform.GetComponent<PlayerCam>().sensY;
+    }
+
+    private void Update()
+    {
+        if (Input.GetMouseButtonDown(0))
+        {
+            //print("Left Click");
```

Comment:
Remove Prints

Reply:
those were jacky's prints so idk if he wants them removed or not

Reply:
he forgot to remove them last time, and we were short on time so we just approved it

Reply:
removed